-- Fix PEP 621 license field for modern setuptools.
-- Upstream pyproject.toml uses the legacy string form `license = "MIT"`,
-- which current setuptools rejects. Convert it to the table form expected
-- by PEP 621.

--- pyproject.toml.orig	2026-07-30 06:35:05 UTC
+++ pyproject.toml
@@ -4,7 +4,7 @@
 description = "aiomisc - miscellaneous utils for asyncio"
 authors = [{ name = "Dmitry Orlov", email = "me@mosquito.su" }]
 readme = "README.rst"
-license = "MIT"
+license = {text = "MIT"}
 requires-python = ">=3.11"
 classifiers = [
     "Development Status :: 5 - Production/Stable",
